test/support/piano.js

```diff
@@ -1,10 +1,10 @@
 
```

Comment:
Hi, @dfcreative 

The `test/support/piano.js` must have double quotes to pass the test. You should revert this changes and find a way to skip standard validation (https://github.com/feross/standard#how-do-i-ignore-files)

Thanks for the pull request :+1: